server/trillian_log_signer/Dockerfile

```diff
@@ -8,8 +8,11 @@ ENV DB_USER=test \
 ENV HOST=0.0.0.0 \
     HTTP_PORT=8091
 
-ENV SEQUENCER_GUARD_WINDOW=0s \
-    FORCE_MASTER=true
+ENV SEQUENCER_GUARD_WINDOW=0s
```

Comment:
Docker creates a new "layer" for each command in the docker file. 
If you group the commands (via `\`) you can save a bunch of time when building.